lib/streamlit/DeltaGenerator.py

```diff
@@ -308,6 +308,13 @@ def _cursor(self):
         else:
             return self._provided_cursor
 
+    def _get_coordinates(self):
+        """Returns the element's 4-component location as string like "M.(1,2).3"."""
+        container = self._container  # proto index of container (e.g. MAIN=1)
+        path = self._cursor.path  # [uint, uint] - "breadcrumbs" w/ ancestor positions
+        index = self._cursor.index  # index - element's own position
+        return "{}.{}.{}".format(container, path, index)
```

Comment:
Looks good. Just some nitpicks:

* Please explain why this is useful in the docstring. Mention the MediaFileManager, etc. I know this function is not specific to the MFM and in the future could be used for other purposes, but seems like the cleanest place to talk about this right now.
* Please capitalize the comment "Proto index (...)" instead of "proto index (...)".

Reply:
Sure